app/src/shared/components/BackButton/index.jsx

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React from 'react'
+import cx from 'classnames'
+import { Translate } from 'react-redux-i18n'
+
+import Link from '../Link'
+import List from '../List'
+import Text from '../Text'
+import SvgIcon from '../SvgIcon'
+import routes from '$routes'
+import styles from './backButton.pcss'
+
+type Props = {
+    className?: string,
+    href?: string,
+    to?: string,
+}
+
+const BackButton = ({ className, ...props }: Props) => (
+    <div
+        className={cx([
+            styles.root,
+            className,
+            Text.styles.uppercase,
+            Text.styles.spaced,
+            Text.styles.plexMono,
+        ])}
+    >
+        &zwnj;
+        <List
+            className={styles.list}
+        >
+            <li>
+                <Link
```

Comment:
Out of scope perhaps here but maybe we'll need to modify the links if we use this component somewhere else (passing them as props).